ui/app/adt/index.html

```diff
@@ -128,8 +142,6 @@
         <script src="../common/concept-set/directives/obsConstraints.js"></script>
         <script src="../common/concept-set/services/conceptSetServices.js"></script>
         <script src="../common/concept-set/services/conceptSetUiConfigService.js"></script>
-        <script src="../common/concept-set/services/conceptGroupFormatService.js"></script>
```

Comment:
@Arjun-Go @binduak If this change was intentional, could you please explain the rationale behind it?